test/route-define-test.js

```diff
@@ -103,7 +104,8 @@ if (typeof steal !== "undefined") {
         mockRoute.start();
 
 
-        canRoute.start();
+				canRoute.data = new DefineMap();
+				canRoute.start();
```

Comment:
The spacing is completely messed up in this file. I’ll submit a separate PR that runs everything through Prettier.